lib/Core/formattedPropertyValue.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+/*global require*/
+var markdownToHtml = require('../Core/markdownToHtml');
+
+function numberWithCommas(x) {
+    var str = x.toString();
+    var idx = str.indexOf('.');
+    var frac = '';
+    if (idx !== -1) {
+        frac = str.substring(idx);
+        str = str.substring(0, idx);
+    }
+    if (str.length < 6) {
+        return str+frac;
+    }
+    return str.replace(/\B(?=(\d{3})+(?!\d))/g, ",") + frac;
+}
+
+/**
+* Format the value for the description, used by the Feature Info Panel.
+* Strings have markdown applied to them, and numbers are formatted with commas.
+* Anything else is returned as-is.
+*
+* @param {} value The value to format.
+*
+*/
+function formattedPropertyValue(value) {
```

Comment:
Another nitpicky thing: how about `formatPropertyValue` (format acts like a verb) instead of `formattedPropertyValue` (formatted acts like an adjective).  Functions do things, so they should be verbs.